styles/media.scss

```diff
@@ -0,0 +1,95 @@
+@import "variables.scss";
+
+@import "mixin.scss";
+
+@media (max-width: 1000px) {
+  .header {
+    &__nav {
+      display: none;
+    }
+    &__social-links {
+      display: none;
+    }
+    &__burger-menu {
+      display: block;
+      width: 40px;
+    }
+  }
+  .home__title {
+    padding: 20px 0 50px;
+  }
+
+  .process {
+    flex-direction: column;
+
+    &__item {
+      width: 100%;
+      padding: 50px 0;
+
+      &:hover {
+        padding: 50px 30px;
```

Comment:
add here 
`display: none`

Hover on a mobile device doesn't work at all. So you can don't display it.